src/MongoDB.Driver.Core/Core/Operations/CreateIndexesOperation.cs

```diff
@@ -124,7 +122,11 @@ public BsonDocument Execute(IWriteBinding binding, CancellationToken cancellatio
             using (var channel = channelSource.GetChannel(cancellationToken))
             using (var channelBinding = new ChannelReadWriteBinding(channelSource.Server, channel, binding.Session.Fork()))
             {
-                var operation = CreateOperation(channel.ConnectionDescription.ServerVersion);
+                var operation = new CreateIndexesUsingCommandOperation(_collectionNamespace, _requests, _messageEncoderSettings)
```

Comment:
I would keep the CreateOperation helper method so we can continue to share code between the Sync and Async Execute methods.

Reply:
And so that we can test creating the operation (see comment in test file).

Reply:
Done.